add KILL statement can kill DDL jobs describe (#16124)
pingcap/docs: sql-statements/sql-statement-admin-cancel-ddl.md

> **Note:**
>

> > - Before v6.2.0, only this operation can cancel DDL jobs, and all other operations and environment changes (such as machine restart and cluster restart) cannot cancel these jobs. Starting from v6.2.0, [`KILL`](/sql-statements/sql-statement-kill.md) statement can also be used to cancel DDL jobs by killing them. 

Reviewer: ```suggestion
> - Before v6.2.0, only this operation can cancel DDL jobs, and all other operations and environment changes (such as machine restart and cluster restart) cannot cancel these jobs. Starting from v6.2.0, [`KILL`](/sql-statements/sql-statement-kill.md) statement can also be used to cancel ongoing DDL jobs by killing them. 
```

> > - Before v6.2.0, only this operation can cancel DDL jobs, and all other operations and environment changes (such as machine restart and cluster restart) cannot cancel these jobs. Starting from v6.2.0, [`KILL`](/sql-statements/sql-statement-kill.md) statement can also be used to cancel ongoing DDL jobs by killing them. 

Reviewer 2: ```suggestion
> - Before v6.2.0, only this operation can cancel DDL jobs, and all other operations and environment changes (such as machine restart and cluster restart) cannot cancel these jobs. Starting from v6.2.0, [`KILL`](/sql-statements/sql-statement-kill.md) statements can also be used to cancel ongoing DDL jobs by killing them. 
```

> - Only this operation can cancel DDL jobs. All other operations and environment changes (such as machine restart and cluster restart) cannot cancel these jobs.
> - This operation can cancel multiple DDL jobs at the same time. You can get the ID of DDL jobs using the [`ADMIN SHOW DDL JOBS`](/sql-statements/sql-statement-admin-show-ddl.md) statement.
> - If the jobs you want to cancel are finished, the cancellation operation fails.

> > - Starting from v6.2.0, [`KILL`](/sql-statements/sql-statement-kill.md) statement can kill DDL jobs. 

Reviewer 3: ```suggestion
> - Starting from v6.2.0, [`KILL`](/sql-statements/sql-statement-kill.md) statement is able to kill DDL jobs. 
```

> > - Only this operation can cancel DDL jobs. All other operations and environment changes (such as machine restart and cluster restart) cannot cancel these jobs.
> > - This operation can cancel multiple DDL jobs at the same time. You can get the ID of DDL jobs using the [`ADMIN SHOW DDL JOBS`](/sql-statements/sql-statement-admin-show-ddl.md) statement.
> > - If the jobs you want to cancel are finished, the cancellation operation fails.
> > - Starting from v6.2.0, [`KILL`](/sql-statements/sql-statement-kill.md) statement is able to kill DDL jobs. 

Reviewer: ```suggestion
> - Before v6.2.0, only this operation can cancel DDL jobs, and all other operations and environment changes (such as machine restart and cluster restart) cannot cancel these jobs. Starting from v6.2.0, [`KILL`](/sql-statements/sql-statement-kill.md) statement can also be used to cancel DDL jobs by killing them. 
> - This operation can cancel multiple DDL jobs at the same time. You can get the ID of DDL jobs using the [`ADMIN SHOW DDL JOBS`](/sql-statements/sql-statement-admin-show-ddl.md) statement.
> - If the jobs you want to cancel are finished, the cancellation operation fails.
```
